elements/pfe-button/src/pfe-button.scss

```diff
@@ -0,0 +1,138 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: button;
+
+:host {
+  // Example local variable, using BEM format
+  // --pfe-button--Property: value;
+  display: inline-block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-variant="primary"]) button,
+button {
+  background-color: #06c;
+  color: #fff;
+  font-size: 16px;
+  padding: 8px 16px;
+  cursor: pointer;
+  border-radius: 3px;
+  border: 0;
+  position: relative;
+
+  &:after {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    content: "";
+  }
+
+  &:hover,
+  &:focus {
+    background-color: #004080;
+  }
+}
+
+:host([pfe-variant="secondary"]) {
+  button {
+    background-color: transparent;
+    color: #06c;
+
+    &:after {
+      border: 1px solid #06c;
```

Comment:
CSS var for width and color?

Reply:
@wesruv I've added a var for color. Will this work? Or should I specify the width separately too?

```
$variables: (
  ...
  Border: 0,
  Border--after: 1px solid,
  BorderColor--after: transparent,
  BorderRadius: #{pfe-var(surface--border-radius)},
  ...
);
```